src/ImageSharp/Processing/Processors/Drawing/DrawImageProcessor.cs

```diff
@@ -15,19 +15,22 @@ public class DrawImageProcessor : IImageProcessor
     /// Initializes a new instance of the <see cref="DrawImageProcessor"/> class.
     /// </summary>
     /// <param name="image">The image to blend.</param>
-    /// <param name="location">The location to draw the blended image.</param>
+    /// <param name="backgroundLocation">The location to draw the foreground image on the background.</param>
+    /// <param name="foregoundRectangle">The rectangular portion of the foreground image to draw.</param>
     /// <param name="colorBlendingMode">The blending mode to use when drawing the image.</param>
     /// <param name="alphaCompositionMode">The Alpha blending mode to use when drawing the image.</param>
     /// <param name="opacity">The opacity of the image to blend.</param>
     public DrawImageProcessor(
         Image image,
-        Point location,
+        Point backgroundLocation,
```

Comment:
we should add an overload with out the `ForegroundRectangle` for back compatability, as this will be an binary breaking change.

Reply:
I always forget these are public.